types.go

```diff
@@ -332,7 +332,9 @@ func getFieldAsString(field reflect.Value) (str string, err error) {
 			}
 		default:
 			// Not a native type, check for marshal method
+			fmt.Printf("Calling marshal, Type: %s, CanInterface: %t\n", field.Kind(), field.CanInterface())
```

Comment:
Should remove these comments I guess

Reply:
How embarrassing. Thanks for catching those; I've removed them.